crypto/ts/__tests__/AccQueue.test.ts

```diff
@@ -346,7 +346,6 @@ describe('AccQueue', () => {
                 })
 
                 it('Should fail if there are 0 leaves', () => {
-                    expect.assertions(1)
```

Comment:
Just curious, why was this removed?

Reply:
I'm not really sure these are needed tbf. You can use expect.assertions when there might be cases of certain assertions are not called, and you want to constraint it to actually check those run. In this case it seems random to me.. For instance:

```ts 
   expect.assertions(plaintext.length)
   for (let i = 0; i < plaintext.length; i++) {
                expect(plaintext[i] !== ciphertext[i + 1]).toBeTruthy()
    }
```
we are looping for plaintext.length iterations, so it's kinda obvious that it will call expect for plaintext.length times

On the other hand, if you had this (very simple and dumb example): 

```ts
 const data = fetchData();
  if (data) {
        expect(data).toBe('expected data');
 }
```
here it would make sense to add expect.assertions(1) -> if you are running the test cases assuming that `data` should always be defined

Reply:
thanks for the explaination! Well as you said there is no meaning of using `expect.assertion()` in this code block.

Reply:
Anyway we don't use jest anymore so it doesn't matter at all